README.md

```diff
@@ -34,7 +34,7 @@ This application for free retrieves, notifies and shows you the latest Apple too
 
 1. Open Terminal.
 
-2. Execute command `brew cask install toolreleases`.
+2. Execute command `brew install cask toolreleases`.
```

Comment:
```suggestion
2. Execute command `brew install --cask toolreleases`.
```

Reply:
By looking at the Homebrew documentation, seems that this is the preferred way how to install Cask's now. :)

Reply:
I took the initiative and added the changes.